client/README.md

```diff
@@ -1,36 +1,64 @@
 # BigQuery Migration Service Batch SQL Translator CLI
 
-The [BigQuery Migration Service][BQMS] includes a product known as the
-[batch SQL translator][batch SQL translator] that supports translation of a
-wide variety of SQL dialects to Google Standard SQL. This command line tool
-simplifies the process of using the batch SQL translator by providing a
-framework for pre and postprocessing of SQL files as well as upload/download of
-SQL files to/from GCS.
+This directory contains the Batch SQL Translator CLI, a command line client
+that simplifies the process of using the [batch SQL translator][batch SQL translator]
+of the [BigQuery Migration Service][BQMS] to translate from a wide variety of
+SQL dialects to Google Standard SQL. This client provides a framework that
+handles pre and postprocessing of SQL files, upload/download of SQL files
+to/from GCS, and invoking the GCP SQL translation API.
+
+Read in the Google Cloud documentation [how to submit a translation job using
+this client][how to submit].
 
-- [Quickstart](#quickstart)
 - [Installation](#installation)
-- [Basic Usage](#basic-usage)
+- [Quickstart](#quickstart)
+- [Running Using `run.sh`](#running-using-runsh)
+- [Running Using `bqms-run` Directly](#running-using-bqms-run-directly)
   - [Environment Variables](#environment-variables)
-  - [config.yaml](#configyaml)
-  - [Recommended Usage](#recommended-usage)
+- [config.yaml](#configyaml)
 - [Metadata Lookup](#metadata-lookup)
   - [DDL and Metadata Dump](#ddl-and-metadata-dump)
   - [Unqualified References](#unqualified-references)
 - [Pre and Postprocessing](#pre-and-postprocessing)
   - [Handling Macro/Templating Languages](#handling-macrotemplating-languages)
+  - [Extraction of Heredoc SQL Statements from KSH Inputs](#extraction-of-heredoc-sql-statements-from-ksh-inputs)
   - [Custom Pre and Postprocessing Logic](#custom-pre-and-postprocessing-logic)
 - [Renaming SQL Identifiers](#renaming-sql-identifiers)
 - [Deploying to Cloud Run](#deploying-to-cloud-run)
 - [Performance](#performance)
 
-## Quickstart
+## Installation
+
+Preferred OS: Linux or macOS. Windows usage may be possible through PowerShell,
+but it is not officially supported.
+
+**Download and extract the latest release zip `dwh-migration-tools-vX.X.X.zip`
+from [the Releases page](https://github.com/google/dwh-migration-tools/releases).**
```

Comment:
Why do we want to enforce users to download zip from the release for this python client?  Have we seen any versioning issues for python client before?

Reply:
There are at least three reasons to want to point users at the releases rather than telling them to `git clone` the source repository:

1. We have had a nontrivial number of users contact our support because they were having issues getting `git clone` to work. Telling users to just download a release short-circuits that entire problem.
2. With the Java dumper, we have had many users contact our support because they were failing to build the source code themselves while apparently being unaware that a binary build was available because it was not clearly mentioned in the documentation. While we do not currently have any comparable build activities for the Python client that would motivate wanting users to download a release over using the GitHub source, directing users to download releases future-proofs these customers' usage workflows against any such build processes we introduce to the client release packaging in the future.
3. When communicating with users about changes to the client, or determining whether a user is using a recent-enough client version to have a certain feature, it is easier to refer to a release version number than to a GitHub hash or download date.